charters/UploadService/charter.md

```diff
@@ -0,0 +1,78 @@
+# Upload Service
+
+## Description
+The Upload Service enables fast, cost-effective upload, staging, and validation of data submitted to the Human Cell Atlas (HCA) Data Coordination Platform (DCP) from across the world.
+
+## Objectives
+1. Enable submitters to upload files quickly and cost-effectively to temporary *staging areas* for subsequent submission to the Data Storage Service (DSS)
+1. Provide tools a command line interface (CLI) and APIs to upload files to staging areas
+1. Provide DCP developers the ability to validate files loaded into staging areas with user-defined validation jobs
```

Comment:
DCP developers aren't really the actors here. I'd suggest "Provide a framework for validating files..."